aiidalab_widgets_base/computational_resources.py

```diff
@@ -199,24 +199,21 @@ def refresh(self, _=None):
     def _validate_value(self, change):
```

Comment:
I think we are misusing traitlet's validation mechanism here. The validation should not extend beyond basic checks IMO. In this case it would be a check whether the string is actually a valid UUID.

Then you can have a second trait, that actually contains the code which is None or set to a magic value in case that the requested code does not exist.

Reply:
Good point, I didn't check the logic carefully. I think in the validate function, we only need to check the UUID is valid value pointer to the node that exists in DB.